requirements.txt

```diff
@@ -1,14 +1,12 @@
-asgiref==3.2.10
```

Comment:
Hi @jackton1 why did you remove this requirement?

Reply:
@jtemporal Was a mistake should be resolved now.